containers/base/Dockerfile

```diff
@@ -181,77 +192,69 @@ RUN echo "deb-src http://ftp.us.debian.org/debian testing main" >> /etc/apt/sour
         --disable-installation-options && \
     tools/google-cloud-sdk/bin/gcloud -q components update \
         gcloud core bq gsutil compute preview alpha beta && \
-    # disable the gcloud update message
+# Disable the gcloud update message
     tools/google-cloud-sdk/bin/gcloud config set component_manager/disable_update_check true && \
     touch /tools/google-cloud-sdk/lib/third_party/google.py && \
-
-# Set our locale to en_US.UTF-8.
-    apt-get install -y locales && \
-    locale-gen en_US.UTF-8 && \
-    update-locale LANG=en_US.UTF-8 && \
-
 # Add some unchanging bits - specifically node modules (that need to be kept in sync
 # with packages.json manually, but help save build time, by preincluding them in an
 # earlier layer).
     /tools/node/bin/npm install \
-        ws@1.1.4 \
+        bunyan@1.7.1 \
         http-proxy@1.13.2 \
         mkdirp@0.5.1 \
+        node-cache@3.2.0  \
         node-uuid@1.4.7 \
-        bunyan@1.7.1 \
         tcp-port-used@0.1.2 \
-        node-cache@3.2.0 && \
+        ws@1.1.4 && \
     cd / && \
     /tools/node/bin/npm install -g forever && \
-
 # Clean up
     apt-get autoremove -y && \
     rm -rf /var/lib/apt/lists/* && \
     rm -rf /tmp/* && \
     rm -rf /root/.cache/* && \
     rm -rf /usr/share/locale/* && \
-    rm -rf /usr/share/i18n/locales/* && \
-    cd /
+    rm -rf /usr/share/i18n/locales/*
```

Comment:
Try to combine all of the above RUN statements into a single RUN statement, including this cleanup.

The reason for this is that otherwise the intermediate output diffs of each RUN are saved as their own docker images, and the overall size of the container increases.

Reply:
Done.